wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/CreateGroupConversationActionHandler.swift

```diff
@@ -21,7 +21,7 @@ import WireDataModel
 
 public final class CreateGroupConversationAction: EntityAction {
 
-    public typealias Result = NSManagedObjectID
+    public typealias Result = (conversationId: NSManagedObjectID, participantIds: Set<QualifiedID>)
```

Comment:
Do we really need to pass back the participant ids if they're available in the scope that created the action? Sync the conversation service creates the action and handles the result, it already knows whether it needs to add mls participants and which ones to add.

Reply:
returning only NSManagedObjectID